pkg/utils/arrays.go

```diff
@@ -5,3 +5,10 @@ func Reverse[T any](a []T) {
 		a[i], a[j] = a[j], a[i]
 	}
 }
+
```

Comment:
I would suggest using the stdlib function for that.
https://pkg.go.dev/slices#Reverse

Reply:
Didn't notice the stdlib function. Replaced